### ISSUE TITLE
Voyant Tools Group 4 (late Eliot)

### ISSUE THREAD

**Issue by cmohge1** [SASDigitalHumanitiesTraining/Visualisation#23]
Please post your thoughts and sample visualisations below.

**Comment by aghague** [comment]
Hi Group 4 members, 

Here's what I'm exploring for exercise 1 - spurred by Monday's discussion regarding how Victorians viewed the relationship between science and art. 

![image](https://user-images.githubusercontent.com/74906423/101652204-e388c080-3a35-11eb-983a-0d8c0f180379.png) 

I like how quick it was to gather a few promising insights: 

- "poetry" and "science" have a comparable number of mentions, but the balance shifts from favouring science in the earlier novel (Middlemarch, 1871) to favouring poetry in the latter (DD 1876) 
- "art" and "science" appear in close proximity to each other only once in each novel; in Middlemarch, "science" comes first (i.e., "science and art"); in DD, it's the other way around (i.e., "art and science"); this fits in nicely with the observation above.
- "art" has significantly more mentions throughout, and the number is fairly constant 
- the collocates for all 3 terms cover a wide range of emotions/perspectives, revealing a nuanced approach to all 3 

I'm thinking of seeing if adding "novels" into the mix refines the picture even further - I'll let you know what I find. 

Please feel free to comment on/ critique what I've done so far - and please *do* suggests better ways of going about it if any spring to mind!

**Comment by Pylaemenes** [comment]
I'd be delighted to think about how this sort of approach leads to deep insights—is Middlemarch contextually different to explain the differences?

**Comment by Pylaemenes** [comment]
Voyant crashed on me part way through, but I did do a couple of wide-lens word studies on Middlemarch:
<img width="445" alt="Screen Shot 2020-12-09 at 5 21 09 PM" src="https://user-images.githubusercontent.com/75645297/101696337-fb674100-3a43-11eb-894e-dad4bec32b81.png">
<img width="445" alt="Screen Shot 2020-12-09 at 5 22 25 PM" src="https://user-images.githubusercontent.com/75645297/101696386-0c17b700-3a44-11eb-8df8-d4e98d6857c6.png">

Since I'm interested in Rome generally, I thought I'd see what it would look like from a bird's-eye view. It occupies a central place early and then literally fades away like a memory.
And I wanted to see the collocation of city and country as the text moves forward. Then the thing crashed and was unavailable, so I moved on.

**Comment by Pylaemenes** [comment]
Ok, I got back in and I uploaded two "marginalia" (or scholia) from ancient commentaries from Homer that I'm working on and compared the two:
<img width="513" alt="Screen Shot 2020-12-09 at 5 42 01 PM" src="https://user-images.githubusercontent.com/75645297/101697849-85b0a480-3a46-11eb-9dad-ca4010c44be3.png">

The texts are only about 150 words each, so not a lot can be said at this scale—is it possible to "map" over two texts that are generally thought to be from the same source to compare? I'd be interested in that technology.

From this limited bit we can see that Scholia T includes more genealogical information (personal names), but Scholia B includes more narrative.

**Comment by aghague** [comment]
@Pylaemenes 

> I'd be delighted to think about how this sort of approach leads to deep insights—is Middlemarch contextually different to explain the differences?

It's a long time since I've read any Eliot closely or have thought with any rigour about the specific circumstances in which each novel was written, so I'm almost certainly missing a lot of nuance here; nevertheless, I would argue that this change of focus is very much in keeping with the general direction of travel evident in debates about the role of science and the nature (or purpose) of art both at the time and in the following decades. It is around the 1870s that things started to unravel economically and intellectually for what we now call the mid-Victorians  (loosely speaking, 1850-1870 or even 1875) - although many of them will not have noticed this yet. Again, I'm oversimplifying, but this is when what once seemed for most to be unquestionable technical superiority and ever-increasing prosperity starts to give way on an increasingly large scale to technical obsolescence and financial decline. Science can do may things, but it is not always very good at providing solace in times of uncertainty and diminishing agency; art, on the other hand, often excels at this, and thus becomes emotionally more relevant. 

It was interesting for me to see echoes of this affective shift woven deep into these texts – whether by accident or by design. I always knew the continuity of theme was there, but it was very satisfying to see it both close up  (collocations) and at a distance (across the two texts) at the same time. I would not want one without the other – the scale would distort the whole in both cases. Taken together, though, they provide a lot of food for thought.

**Comment by aghague** [comment]
@Pylaemenes 

> Ok, I got back in and I uploaded two "marginalia" (or scholia) from ancient commentaries from Homer that I'm working on and compared the two:
> <img alt="Screen Shot 2020-12-09 at 5 42 01 PM" width="513" src="https://user-images.githubusercontent.com/75645297/101697849-85b0a480-3a46-11eb-9dad-ca4010c44be3.png">
> 
> The texts are only about 150 words each, so not a lot can be said at this scale—is it possible to "map" over two texts that are generally thought to be from the same source to compare? I'd be interested in that technology.
> 

Have you seen Juxta? https://juxtasoftware.org/  Might do what you need.

**Comment by cmohge1** [comment]
Yes, I have used Juxta (I also know the developer, as he worked with me on
the Melville Electronic Library). It can be a powerful collation tool, but
it is not being maintained.

مكتوب
